jsondiff.go

```diff
@@ -725,29 +735,30 @@ func compareAndColorizeMaps(a, b map[string]interface{}, indent string, red, gre
 // expect: The map containing the expected header values.
 // actual: The map containing the actual header values.
 // Returns a ColorizedResponse containing the colorized differences for the expected and actual headers.
-func CompareHeaders(expectedHeaders, actualHeaders map[string]string) Diff {
-	var expectAll, actualAll strings.Builder // Builders for the resulting strings.
+func CompareHeaders(expectedHeaders, actualHeaders map[string]string, options ...bool) Diff {
+	disableColor := false
+	if len(options) > 0 {
```

Comment:
better to keep it array of interfaces no? and define in comments as well

Reply:
<img width="1106" alt="Screenshot 2024-10-21 at 3 34 22 PM" src="https://github.com/user-attachments/assets/3735dfc1-b187-4622-aa2c-f36132c2bb07">

did you mean this when you said to use array of interfaces?

Reply:
Yes

Reply:
yes sir added the changes